offline.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Status Offline
+description: "Page not found. Your pixels are in another canvas."
+permalink: /offline.html
+---
+
+The tumbeasts are to be blamed, as it seems you have been voted off the Internet.
+
+This site makes use of a [service worker](https://developers.google.com/web/fundamentals/getting-started/primers/service-workers), which is probably why you can actually see this page.
+
+<figure>
+  <amp-img src="{{ site.url }}/assets/images/tbrun1.png"
```

Comment:
A custom offline page, and let's face it, [tumbeasts](http://knowyourmeme.com/memes/the-tumbeasts) are awesome.